### ISSUE TITLE
Error when using "Open with..." on a py2exe program

### ISSUE THREAD

**Issue by Ragster2448** [albertosottile/py2exe#48]
I'm trying to make my program built with py2exe utilize the Windows "Open with..." function. Running the program alone works fine, but when I try to run the program on a file using "Open with..." I get this error:

```
Traceback (most recent call last):
  File "amkeditor.pyw", line 993, in <module>
  File "tkinter\__init__.pyc", line 2023, in __init__
_tkinter.TclError: Can't find a usable init.tcl in the following directories: 
    {C:\Users\marag\Downloads\lib\tcl} C:/Users/marag/Downloads/lib/tcl8.6 {C:/Users/marag/Documents/Python Projects/AMK Editor/lib/tcl8.6} {C:/Users/marag/Documents/Python Projects/AMK Editor/lib/tcl8.6} {C:/Users/marag/Documents/Python Projects/lib/tcl8.6} {C:/Users/marag/Documents/Python Projects/AMK Editor/library} {C:/Users/marag/Documents/Python Projects/library} {C:/Users/marag/Documents/Python Projects/tcl8.6.9/library} C:/Users/marag/Documents/tcl8.6.9/library



This probably means that Tcl wasn't installed properly.
```

It looks like it's trying to find the "lib/tcl" folder, which is located in the same directory as the program, but for some reason it is searching the directory where the file that I used the "Open with..." function on instead. None of the paths in the error are where the "lib/tcl" folder is actually located, which is "C:/Users/marag/Documents/Python Projects/AMK Editor/dist/lib/tcl" Here is my py2exe setup.py:

```
from distutils.core import setup
import py2exe, sys, os

sys.argv.append('py2exe')

setup(
    options = {'py2exe': {'includes': ['tkinter'], 'bundle_files': 2, 'compressed': True}},
    windows = [{'script': 'amkeditor.pyw', 'icon_resources': [(1, 'amkeditor.ico')]}],
    zipfile = None
)
```

As I said before, this happens ONLY when using "Open with...". The program works correctly when opened alone.

**Comment by albertosottile** [comment]
Thanks for reporting this, I can imagine why this is happening: https://github.com/albertosottile/py2exe/blob/b6dfefd6879061d7ea99385cad252f955708fa9b/py2exe/hooks.py#L296

However, I have to think about how to fix the bootloader hook code to be resilient to this kind of path changes.

**Comment by albertosottile** [comment]
This should be fixed in 0.10.0.2.
